config/config/common-config.yaml

```diff
@@ -818,3 +818,13 @@ rookCeph:
     enabled: false
   gatekeeperPsp:
     enabled: false
+
+nodeLocalDns:
+  customConfig: #|
+    #example.example:example {
+    #    errors
+    #    cache 30
+    #    reload
+    #    loop
+    #    forward . __PILLAR__UPSTREAM__SERVERS__
+    #    }
```

Comment:
Would be nice with some more verbose examples here, like what the first and second "arguments" of "forward" means, as well as "cache x".

Reply:
Added a link to the docs which explains the options nicely :+1: